Sources/ComposableArchitecture/Effects/Throttling.swift

```diff
@@ -27,23 +27,27 @@ extension Effect {
         return Observable.just(value)
       }
 
-      guard
-        scheduler.now.timeIntervalSince1970 - throttleTime.timeIntervalSince1970
-          < interval.timeInterval
-      else {
+      let value = latest ? value : (throttleValues[id] as! Output? ?? value)
```

Comment:
This forced downcast matches the upstream implementation so I will 👍 it, but it is weird. I wish there were a better pattern than this…!

Reply:
100% agree.  The only reason I didn't replace the forced downcast for something a little less all-or-nothing was that it'd then make it harder for us to easily compare our implementation to the upstream repo if/when they were to make updates on their end.